src/System.Runtime/ref/System.Runtime.cs

```diff
@@ -3137,6 +3137,7 @@ public partial class ExternalException : System.SystemException
         public ExternalException(string message, int errorCode) { }
         protected ExternalException(System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context) { }
         public virtual int ErrorCode { get; }
+        public override string ToString() { throw null; }
```

Comment:
> Do we need to include overrides in the contract?

Ideally, yes.  Otherwise there are situations where a derived type using base.Whatever could end up skipping the base. cc: @weshaggard